Notebooks/Pluto/a1.jl

```diff
@@ -13,7 +13,7 @@ md"# Assignment Sheet 1"
 # ╔═╡ 1002aa7c-05ac-11eb-3b42-87809ac716c4
 md" **_Assignment 1_:** 
 1. _Make the following circuit in Yao_.
-2. _Make the qubits with state \"0000\", and feed it to the circuit._ Hint: You can use either of zero_state(number of qubits) function or ArrayReg(bit\" \") function
+2. _Make the qubits with state $ “0000” $, and feed it to the circuit._ Hint: You can use either of ` zero_state(number of qubits) ` function or ` ArrayReg(bit\" \") ` function
```

Comment:
I think we should use a ket here if it's gonna be a math expression?

Reply:
and in case you don't know this, in Julia we have a Julia flavored markdown format where we use ` ``<math expr>`` ` to represent an inline math expr. because $ can be easily mixed up with the string interpolation mark. Would be nice if we can be consistent on this in the tutorial.

Reply:
I don't recommend showing Yao code to the reader in the first 3 notebooks.
That'll just confuse them😅
The use of bits" " is properly shown later.

Reply:
Thanks！Didn't know this！Will look into this！